wrappers/rust/icicle-core/src/ntt/mod.rs

```diff
@@ -87,6 +100,7 @@ pub trait NTT<F: FieldImpl> {
     ) -> IcicleResult<()>;
     fn initialize_domain(primitive_root: F, ctx: &DeviceContext) -> IcicleResult<()>;
     fn get_default_ntt_config() -> NTTConfig<'static, F>;
+    fn get_default_ntt_config_for_device(device_id: usize) -> NTTConfig<'static, F>;
```

Comment:
consider adding device_id param to get_default_ntt_config() with default value to avoid having two methods